### ISSUE TITLE
Manual requires fonts not part of build system

### ISSUE THREAD

**Issue by alerque** [sile-typesetter/sile#854]
Currently cutting releases requires Noto Sans CJK, Gentium Book Basic, and Scheherazade which are all readily available with open licenses (also Adobe Caslon Pro, which is not, see #853); but our build system doesn't actually fetch them so manuals are not consistent unless the builder knows to get these fonts first.

The build system should know that building docs requires these.

**Comment by OlivierNicole** [comment]
It even seems to cause the build to fail completely for me, presumably because the expected fallback for a Japanese character is Noto Sans CJK JP and I don’t have it. When doing `make docs`:

```
This is SILE v0.10.4.r9-g2c13d78-dirty
<documentation/developers.sil>
<documentation/macros.xml>

! Font family 'Helvetica Neue' not available, falling back to 'Libertinus Serif' at documentation/macros.xml:9:67: in \par
[1] [2] 
! Font family 'Gentium Book Basic' not available, falling back to 'Cantarell' at documentation/macros.xml: in \hbox near <./packages/tableofcontents.lua:78>:6:90: in \process
[0] [1] [2] [3] [4] 
Harfbuzz version: 2.6.4
Shapers enabled: graphite2, ot, fallback
ICU support enabled

Fonts used:
/home/olivier/.local/share/fonts/Garamond-Math.otf:0	Version 2019-02-05 
documentation/Silson-Condensed.otf:0	Version 1.001;PS 001.001;hotconv 1.0.88;makeotf.lib2.5.64775
/usr/share/fonts/TTF/GentiumPlus-R.ttf:0	Version 5.000
/home/olivier/.local/share/fonts/Garamond-Math.otf:0	Version 2019-02-05 
/home/olivier/.local/share/fonts/Garamond-Math.otf:0	Version 2019-02-05 
/usr/share/fonts/cantarell/Cantarell-ExtraBold.otf:0	Version 0.201
/usr/share/fonts/OTF/LibertinusSerif-Semibold.otf:0	Version 6.10 
/usr/share/fonts/OTF/LibertinusSans-Italic.otf:0	Version 6.10 
/usr/share/fonts/TTF/DejaVuSansMono.ttf:0	Version 2.37
/home/olivier/.local/share/fonts/Garamond-Math.otf:0	Version 2019-02-05 
documentation/Silson-Condensed.otf:0	Version 1.001;PS 001.001;hotconv 1.0.88;makeotf.lib2.5.64775
/usr/share/fonts/TTF/DejaVuSansMono.ttf:0	Version 2.37
/usr/share/fonts/OTF/LibertinusSerif-Semibold.otf:0	Version 6.10 
true
pg0=$(pdfinfo documentation/sile.pdf | awk '$1 == "Pages:" {print $2}' || echo 0) hadtoc=$([[ -f documentation/sile.toc ]] && echo true || echo false)
mkdir -p .deps
./sile -m .deps/documentation/sile.d -d versions -f fontconfig documentation/sile.sil -o documentation/sile.pdf && pg0=${pg} pg=$(pdfinfo documentation/sile.pdf | awk '$1 == "Pages:" {print $2}' || echo 0)
if [[ -f documentation/sile.toc ]] && echo true || echo false; then
${hadtoc} ||  ./sile -m .deps/documentation/sile.d -d versions -f fontconfig documentation/sile.sil -o documentation/sile.pdf && pg0=${pg} pg=$(pdfinfo documentation/sile.pdf | awk '$1 == "Pages:" {print $2}' || echo 0)
[[ ${pg} -gt ${pg0} ]] &&  ./sile -m .deps/documentation/sile.d -d versions -f fontconfig documentation/sile.sil -o documentation/sile.pdf && pg0=${pg} pg=$(pdfinfo documentation/sile.pdf | awk '$1 == "Pages:" {print $2}' || echo 0) ||:
fi
Syntax Error: Couldn't find trailer dictionary
Syntax Error: Couldn't find trailer dictionary
Syntax Error: Couldn't read xref table
This is SILE v0.10.4.r9-g2c13d78-dirty
<documentation/sile.sil>
<documentation/macros.xml>

! Font family 'Gentium Book Basic' not available, falling back to 'Garamond-Math' at documentation/macros.xml: in \par near <./classes/plain.lua:93>:27:25: in \raggedtable: 0x557ef012ad00
[1] [2] [3] [4] [0] [1] [2] [3] [4] [5] 
! Font family 'Adobe Caslon Pro' not available, falling back to 'Garamond-Math' at in " at the top left, …"
[6] [7] [8] [9] [10] [11] [12] [13] [14] [15] 
! Underfull frame: 220.56961170055pt stretchiness required to fill but only 23pt available at documentation/macros.xml:672:1: in \verbatim
[16] [17] [18] [19] 
! Overfull frame: 66.9969703307pt shrinkability required to fit but only 24pt available at documentation/macros.xml: in \par near <./classes/plain.lua:93>:25:30: in \raggedtable: 0x557ef0129fa0
[20] [21] 
! Font family 'Scheherazade' not available, falling back to 'Garamond-Math' at <./classes/plain.lua:93>:14:27: in \penaltytable: 0x557eeee7ccf0
[22] [23] [24] [25] [26] <packages/rules>[27] <packages/color><packages/background><packages/rotate>
! Underfull frame: 67.7999671693pt stretchiness required to fill but only 49pt available at in " degrees.␤␤The…" near documentation/macros.xml:1111:1: in \package-documentationtable: 0x557eeeafb6f0
[28] <packages/features>
! Underfull frame: 57.519401737258pt stretchiness required to fill but only 31pt available at documentation/macros.xml:5.3567960152785e-315:5.4746558889222e-315: in \skiptable: 0x557ef0cb4ff0
[29] <packages/unichar>
! Font family 'Symbola' not available, falling back to 'Garamond-Math' at <./classes/plain.lua:93>:14:27: in \penaltytable: 0x557eeee7ccf0
<packages/bidi><packages/pullquote>[30] [31] 
! Underfull frame: 15.4259671693pt stretchiness required to fill but only 0pt available at in ":␤␤"
[32] <packages/linespacing>[33] <packages/font-fallback>
! Font family 'Noto Sans CJK JP' not available, falling back to 'Source Code Variable' at in "?␤␤Many applic…" near documentation/macros.xml:1216:1: in \package-documentationtable: 0x557eeeb06790

! Some glyph(s) not available in any fallback font, run with '-d font-fallback' for more detail at in "?␤␤Many applic…" near documentation/macros.xml:1216:1: in \package-documentationtable: 0x557eeeb06790

Error detected:
	./packages/font-fallback.lua:115: bad argument #1 to 'max' (value expected)
true
Syntax Error: Couldn't find trailer dictionary
Syntax Error: Couldn't find trailer dictionary
Syntax Error: Couldn't read xref table
```

**Comment by alerque** [comment]
Not having a font should _never_ cause a hard crash. What you are reporting there is not the same as this issue. It might be more related to attempts to fix #716 or perhaps it it a duplicate of #531. The error message is different but SILE has changed a lot since that latter report too. Or perhaps its a different bug.
